dev/com.ibm.ws.org.apache.myfaces.2.3/src/com/ibm/ws/jsf/taglibconfig/package-info.java

```diff
@@ -12,4 +12,4 @@
  * @version 1.0.0
  */
 @org.osgi.annotation.versioning.Version("1.0.0")
-package com.ibm.ws.jsf.cdi;
\ No newline at end of file
+package com.ibm.ws.jsf.taglibconfig;
```

Comment:
Can you explain why we're renaming the cdi package to tabligconfig I thought we were just adding a new taglibconfig package and splitting extprocess/taglibconfig?

Reply:
I commented above about this. 

But here are links to the new package split:

2.3 bundle: https://github.com/OpenLiberty/open-liberty/tree/a309031ce2ff709db516a432ea07bf9c80496ea8/dev/com.ibm.ws.org.apache.myfaces.2.3/src/com/ibm/ws/jsf/taglibconfig

3.0 bundle: https://github.com/OpenLiberty/open-liberty/tree/a309031ce2ff709db516a432ea07bf9c80496ea8/dev/com.ibm.ws.org.apache.myfaces.2.3/src/com/ibm/ws/jsf/taglibconfig

Internal Bundle: https://github.com/OpenLiberty/open-liberty/tree/a309031ce2ff709db516a432ea07bf9c80496ea8/dev/io.openliberty.faces.internal/src/com/ibm/ws/jsf/extprocessor 

The CDI initalization class (JSFExtensionFactory) is shared between two features.

Reply:
Yea I understand this now however it looks as though you're changing cdi -> taglibconfig is this just an odd git issue here? As far as I know you haven't changed anything in the com.ibm.ws.jsf.cdi package have you??

Reply:
ah, that is a a git issue here. I had simply removed _WASCDIAnnotationInjectionProvider.java_ since it's in the io.openliberty.faces.internal package.  Git must be misinterpreting which package i renamed. 

It really should be `com.ibm.ws.jsf.extprocessor` -> `com.ibm.ws.jsf.taglibconfig`  instead.